src/jit/lowerarm.cpp

```diff
@@ -32,6 +32,91 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 #include "lower.h"
 #include "lsra.h"
 
+// there is not much lowering to do with storing a local but
+// we do some handling of contained immediates and widening operations of unsigneds
+void Lowering::LowerStoreLoc(GenTreeLclVarCommon* storeLoc)
```

Comment:
All of the methods require function headers. See [https://github.com/dotnet/coreclr/blob/master/Documentation/coding-guidelines/clr-jit-coding-conventions.md#94-function-header-comment](https://github.com/dotnet/coreclr/blob/master/Documentation/coding-guidelines/clr-jit-coding-conventions.md#94-function-header-comment)